packages/file-asset-apis/src/s3/createS3Client.ts

```diff
@@ -1,85 +1,145 @@
 import fs from 'fs-extra';
 import { Agent, AgentOptions as HttpsAgentOptions } from 'https';
-import { S3Client as BaseClient } from '@aws-sdk/client-s3';
+import { S3Client } from '@aws-sdk/client-s3';
 import { NodeHttpHandler, NodeHttpHandlerOptions } from '@smithy/node-http-handler';
-import type { S3ClientConfig as baseConfig } from '@aws-sdk/client-s3';
-import {
-    debugLogger, has, isEmpty, isNumber
-} from '@terascope/utils';
-import type { S3Client } from './client-types';
+import type { S3ClientConfig } from '@aws-sdk/client-s3';
+import { debugLogger } from '@terascope/utils';
+// import type { S3Client } from './client-types';
 
-export interface S3ClientConfig extends baseConfig {
+export interface S3ConnectionConfig {
+    endpoint: string,
+    region: string,
+    forcePathStyle?: boolean,
+    bucketEndpoint?: boolean,
+    accessKeyId?: string
+    secretAccessKey?: string,
+    maxRetries?: number,
     sslEnabled?: boolean,
     certLocation?: string,
-    httpOptions?: HttpsAgentOptions,
-    handlerOptions?: Pick<NodeHttpHandlerOptions, 'requestTimeout'|'connectionTimeout'>,
-    secretAccessKey?: string,
-    accessKeyId?: string
+}
+
+export interface Credentials {
```

Comment:
The name of this interface could be more specific ... it's not a username and password, the properties are names specific to S3.  Why not `S3ClientCredentials`?